docs/sites.yml

```
@@ -6850,3 +6850,14 @@
   built_by: Fourpost
   built_by_url: https://github.com/fourpost
   featured: false
+  - title: PersonalBlog Example
```

Can you give it a proper title? PersonalBlog example sounds more like a starter than a website. Are you sure you want to add it to the showcase?
```suggestion
  - title: PersonalBlog Example
```

---

Thank you for your correction, I have changed